test/unit/modules/observer/observer.spec.js

```diff
@@ -316,6 +316,21 @@ describe('Observer', () => {
     }).then(done)
   })
 
+  it('Cannot enable reactivity on a property that is already defined', done => {
+    const vm = new Vue({
```

Comment:
A Vue instance must have a template prop or render fn, your test is throwing an error before getting to the checks

Reply:
@dhensche Thanks; that was it.